functions/api/wb/settings/server/rules.ts

```diff
@@ -0,0 +1,74 @@
+import type { Env } from 'wildebeest/backend/src/types/env'
+import type { ContextData } from 'wildebeest/backend/src/types/context'
+import * as errors from 'wildebeest/backend/src/errors'
+import { type Database, getDatabase } from 'wildebeest/backend/src/database'
+
+export const onRequestGet: PagesFunction<Env, any, ContextData> = async ({ env, request }) => {
+	return handleRequestPost(await getDatabase(env), request)
+}
+
+export async function handleRequestGet(db: Database) {
```

Comment:
Mastodon already has an API to retrieve rules: https://docs.joinmastodon.org/methods/instance/#rules. let's use that instead of a new API.

Reply:
@xtuc I've moved the file so that the endpoint matches Mastodon, but the page you shared only shows the GET, my endpoint included POST and DELETE as well, is that ok?

Reply:
we can use Mastodon's GET and expose our own POST/DELETE endpoint. I'm assuming that's what mastodon does too

Reply:
ok cool thanks 🙂👍